be/src/olap/rowset/beta_rowset.cpp

```diff
@@ -90,7 +90,13 @@ Status BetaRowset::get_inverted_index_size(size_t* index_size) {
                         InvertedIndexDescriptor::get_index_file_path_v1(
                                 InvertedIndexDescriptor::get_index_file_path_prefix(seg_path),
                                 index->index_id(), index->get_index_suffix());
-                RETURN_IF_ERROR(fs->file_size(inverted_index_file_path, &file_size));
```

Comment:
add check in frontend to test the inverted index should only be added for V2 format